staker/type.gno

```diff
@@ -0,0 +1,27 @@
+package staker
+
+import (
+	"std"
+
+	u256 "gno.land/p/demo/uint256"
+)
+
+type IStaker interface {
```

Comment:
what for?

Reply:
It doesn't do anything. I'll delete it

Reply:
https://github.com/gnoswap-labs/governance/pull/4/commits/2e380466ac0fe0d7f67ad7cb5d8b045a9d99e8f8